### ISSUE TITLE
Incorrect version in v0.5.1 tag

### ISSUE THREAD

**Issue by sgerrand** [GoogleCloudPlatform/container-diff#147]
Building this project from the `v0.5.1` tag results in a binary which returns `v0.5.0`. Looks like #142 is the cause.
```
$ git checkout v0.5.1
HEAD is now at 5884ef7... Merge pull request #138 from nkubala/docs
```
```
$ GOPATH=~ make
mkdir -p ./out
GOOS=darwin GOARCH=amd64 CGO_ENABLED=0 go build -tags "container_image_ostree_stub containers_image_openpgp" -ldflags "-X github.com/GoogleCloudPlatform/container-diff/version.version=v0.5.0" -o out/container-diff-darwin-amd64 github.com/GoogleCloudPlatform/container-diff
cp ./out/container-diff-darwin-amd64 out/container-diff
```
```
$ ./out/container-diff
v0.5.0
```

**Comment by sgerrand** [comment]
Can the git tag be recreated?

**Comment by nkubala** [comment]
Hey @sgerrand, sorry about this, as you pointed out I forgot to bump the version in the makefile before I cut the last release. I'm not a huge fan of recreating tags because that can cause people's local forks to get out of sync with the server, so I'll go ahead and just cut a `v0.5.2` release that will have the correct version.

**Comment by sgerrand** [comment]
Fair enough. Thanks for picking this up.

**Comment by sgerrand** [comment]
Resolved by #148. 💝
